src/gui/properties/trackerlistwidget.cpp

```diff
@@ -591,7 +591,7 @@ QStringList TrackerListWidget::headerLabels()
         "#"
         , tr("URL")
         , tr("Status")
-        , tr("Received")
+        , tr("Tracker Peers")
```

Comment:
I wonder if it's just me that think this is even more confusing than "Received". I mean because there is already a Peers column and people will wonder "huh? why there are 2 peer columns and which is which. Which is the one i want?"

Reply:
Maybe `Tracker Connections`? But then this might mislead users into thinking they're connected to all of those peers. Then again, `Seeds` and `Peers` should make it clear how many peers they're connected to.

I think all of this confusion stems from the ambiguous use of `peers` to generally mean other bittorrent clients, and also to mean other users downloading the file (w/ <100% progress).

Reply:
>users downloading the file (w/ <100% progress).

Isn't "leech" mean that?
"Peer" is any client, AFAIK.

Reply:
> Isn't "leech" mean that?

YES, I couldn't think of this word but I think it solves this issue.

I propose the columns should be `peers`, `seeds`, and `leeches`. It should resolve the confusion and is the most "correct."

Reply:
But _peers_ imply all the peers the tracker knows of, doesn't it? Personally i can't think of something short and more descriptive than _Received_ but i'm biased because i already know what it refers to...

Reply:
libtorrent refers to this field as `num_peers`, describing it as "the number of peers this torrent currently is connected to" [[source](https://github.com/arvidn/libtorrent/blob/master/include/libtorrent/torrent_status.hpp#L328)].

Reply:
// tells how many peers the tracker returned in this response. This is
		// not expected to be greater than the ``num_want`` settings. These are not necessarily
		// all new peers, some of them may already be connected.
		int num_peers;

Reply:
Can anyone else chime in on whether they find this confusing? I would like to go ahead with `peers`, `seeds`, and `leeches` but we can revisit if others disagree.

Reply:
I'm OK with peer/seeds/leeches. Thx for this functionality!